tool/src/java/org/sakaiproject/blogwow/tool/producers/BlogViewProducer.java

```diff
@@ -83,12 +85,16 @@ public void fillComponents(UIContainer tofill, ViewParameters viewparams, Compon
         if (!ExternalLogic.NO_LOCATION.equals(externalLogic.getCurrentLocationId())) {
         	navBarRenderer.makeNavBar(tofill, "navIntraTool:", VIEW_ID);
         }
-        List<BlogWowBlog> blogs = new ArrayList<BlogWowBlog>();
+        List<BlogWowBlog> blogs = new ArrayList<>();
         if(params.blogid == null){
-        	//this is a "group" view, so display all available blogs:
-        	blogs = blogLogic.getAllVisibleBlogs(params.locationId, null, true, 0, 0);
+            //this is a "group" view, so display all available blogs:
+            if (serverConfigurationService.getBoolean("blogwow.allow-list-all-blogs", true)) {
```

Comment:
This should be off by default, right?

Reply:
It should be on by default, I think. That's the current behaviour. Didn't want to throw anybody.